packages/note/src/react/index.js

```diff
@@ -0,0 +1,214 @@
+import * as glamor from 'glamor'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Avatar from '@pluralsight/ps-design-system-avatar/react'
+import Icon from '@pluralsight/ps-design-system-icon/react'
+import { useTheme } from '@pluralsight/ps-design-system-theme/react'
+import {
+  arrayOfMaxLength,
+  elementOfType
+} from '@pluralsight/ps-design-system-prop-types'
+import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
+
+import css from '../css/index.js'
+
+const styles = {
+  note: (themeName, props) =>
+    glamor.compose(
+      glamor.css(css['.psds-note']),
+      glamor.css(css[`.psds-note.psds-theme--${themeName}`])
+    ),
+
+  actionBar: themeName =>
```

Comment:
A bit of a change here, with the styles functions not taking "props", instead just `themeName` as a single string.  What was your thought there?

Oh, I'm just seeing the `note: (themeName, props) =>` signature above, also different.

Reply:
now that themeName isn't coming from props but from `useContent` is seemed better to pass it to the styles separate from `props`. i opted for doing `themeName` as the first arg and `props` as the second